dev/RatingControl/RatingControl.cpp

```diff
@@ -833,7 +858,7 @@ double RatingControl::CalculateTotalRatingControlWidth()
     if (captionAsWinRT.size() > 0)
     {
         // TODO MSFT #10030063: Convert to itemspacing DP
```

Comment:
This comment still relevant? Or should it be moved to ItemSpacing()?

Reply:
10030063 is cut but I guess converting to DP is still relevant, moved to ItemSpacing()